### ISSUE TITLE
Transpose in place vs. NOT in place can lose microtones

### ISSUE THREAD

**Issue by gregchapman-dev** [cuthbertLab/music21#1553]
**music21 version**

`9.x` (and probably 8.x and 7.x)   

**Problem summary**

Transpose in place vs. NOT in place can lose microtones.

Here is the suspect code (in Pitch.transpose), but honestly I'm not sure yet if this is actually the culprit.  I will investigate further.

```
        if not inPlace:
            return p
        else:
            # can setName with nameWithOctave to recreate all essential
            # pitch attributes
            # NOTE: in some cases this may not return exactly the proper config
            self.name = p.name
            if self.octave is not None:
                self.octave = p.octave
            # manually copy accidental object
            self.accidental = p.accidental
            # set fundamental
            self.fundamental = p.fundamental
            self.spellingIsInferred = p.spellingIsInferred
            return None
```

**Comment by mscuthbert** [comment]
Yes. If not inPlace then before returning p we should check if self._microtone is not None and if it exists we should deepcopy it.

**Comment by gregchapman-dev** [comment]
I'll go ahead and fix this in PR #1545 (and remove my workaround there as well).

**Comment by gregchapman-dev** [comment]
Oh interesting!  I actually had to do the opposite to fix the bug I was running into:  If inPlace, then we should check if p._microtone is not None and if it exists we should deep copy it to self._microtone. 

I'll see if I can repro the problem you were trying to fix and apply that fix as well.

**Comment by gregchapman-dev** [comment]
I can't make anything go bad with transposition (not inPlace) of pitches with microtones in them.  So I'm going to call this fixed (in PR #1545), and call it a day.
